### ISSUE TITLE
No IFTTT activity on "Notify gpio actions"

### ISSUE THREAD

**Issue by oblivionratula** [vitormhenrique/OctoPrint-Enclosure#262]
I have "Notify gpio actions" checked and have my working Key set in the the Advanced Settings. GPIOs properly toggle when I click the buttons and auto startup and shutdown work, but there is no network traffic to IFTTT and setting the enclosure plugin log to DEBUG only shows the GPIO activity but no indication it's trying to communicate w/ IFTTT at all.

**Comment by vitormhenrique** [comment]
Did you also setup a custom hook on the IFTTT  using the template that I
have documented (somewhere) ?

**Comment by oblivionratula** [comment]
Yes, I did. As I said, there isn't even any network traffic going out towards IFTTT, which I can clearly see generated by the IFTTT plugin itself.

**Comment by oblivionratula** [comment]
<img width="701" alt="Screenshot 2019-10-10 15 45 49" src="https://user-images.githubusercontent.com/9834921/66605326-85342980-eb75-11e9-8d93-37e4c570ee7e.png">
----------------------------------------
<img width="561" alt="Screenshot 2019-10-10 15 45 17" src="https://user-images.githubusercontent.com/9834921/66605327-85342980-eb75-11e9-930e-57ed2193986d.png">
